### PR TITLE
Update Katalon developers

### PR DESCRIPTION
**Pull request by devalex88**
# Description

Github repository: https://github.com/jenkinsci/katalon-plugin
Hosting request: https://github.com/jenkins-infra/repository-permissions-updater/pull/960
Maintainer: @devalex88 
Newly added: @hainguyenkatalon, @duyluonganh

# Submitter checklist for adding or changing permissions

### Always

- [x] Add link to plugin/component Git repository in description above

### For a newly hosted plugin only

- [x] Add link to resolved HOSTING issue in description above

### For a new permissions file only

- [x] Make sure the file is created in `permissions/` directory
- [x] `artifactId` (pom.xml) is used for `name` (permissions YAML file).
- [x] [`groupId` / `artifactId` (pom.xml) are correctly represented in `path` (permissions YAML file)](https://github.com/jenkins-infra/repository-permissions-updater/#managing-permissions)
- [x] Check that the file is named `plugin-${artifactId}.yml` for plugins

### When adding new uploaders (this includes newly created permissions files)

- [x] [Make sure to `@`mention an existing maintainer to confirm the permissions request, if applicable](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)
- [x] Use the Jenkins community (LDAP) account name in the YAML file, not the GitHub account name
- [x] Make sure to `@`mention the users being added so their GitHub account names are known if they require GitHub merge access (see below).
- [x] [All newly added users have logged in to Artifactory at least once](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)

### Reviewer checklist (not for requesters!)

- [x] Check this if newly added person also needs to be given merge permission to the GitHub repo (please @ the people/person with their GitHub username in this issue as well). If needed, it can be done using an [IRC Bot command](https://jenkins.io/projects/infrastructure/ircbot/#github-repo-management)
- [x] Check that the `$pluginId Developers` team has `Admin` permissions while granting the access.
- [ ] In the case of plugin adoption, ensure that the Jenkins Jira default assignee is either removed or changed to the new maintainer.
- [ ] If security contacts are changed (this includes add/remove), ping the security officer (currently `@Wadeck`) in this pull request. If an email contact is changed, wait for approval from the security officer.

There are [IRC Bot commands](https://jenkins.io/projects/infrastructure/ircbot/#issue-tracker-management) for it

### CODE REVIEW

File: permissions/plugin-katalon.yml [hunk 1]
```diff
@@ -6,4 +6,4 @@ issues:
 paths:
 - "org/jenkins-ci/plugins/katalon"
 developers:
-- "devalex88"
+- "hainguyenkatalon,duyluonganh,HieuBui419"
```
Comment (halkeye) [inline permissions/plugin-katalon.yml:9]: this is a yaml list, so should be
```suggestion
- "hainguyenkatalon"
- "duyluonganh"
- "HieuBui419"
```

**Comment by halkeye** [comment]
You also need to check the approprate checkboxes in the pull request template, and cc the existing maintainer.

**Comment by devalex88** [comment]
Thank you @halkeye. I've updated PR according to your comment.

**Comment by halkeye** [comment]
> Use the Jenkins community (LDAP) account name in the YAML file, not the GitHub account name

Did you do this?

> [All newly added users have logged in to Artifactory at least once](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)

Or this? The build is failing cause  hainguyenkatalon isn't in the list of artifactory users

**Comment by devalex88** [comment]
@halkeye He did that yesterday. I'm sorry I think it's because I had saved the file before that.

image

**Comment by halkeye** [comment]
Looks like there's a backlog for running the CI job.

Considering its not yet approved by the original maintainer its not really in a rush.

**Comment by halkeye** [comment]
I do see them in https://reports.jenkins.io/artifactory-ldap-users-report.json so that part should at least pass

**Comment by devalex88** [comment]
It seems all checks have passed. Please let me know if there is anything else needed. Thank you all!

**Review by timja — APPROVED** [review]

**Comment by devalex88** [comment]
Thank you @timja! When would new developers be able to work on the plugin repository?

**Comment by timja** [comment]
now? they need to accept the invite at https://github.com/jenkinsci

**Pull request merged**